float32[] der_robot
float32[] der_target
float32[] kec_robot
float32[] kec_target
float32[] a_robot
float32[] a_target